extension/content/opt-out-ext.js

```diff
@@ -1,104 +1,102 @@
 let selector;
+let option = 'tw';
+
 const root = document.getElementById('doc') || document.getElementById('react-root');
 
 if (document.querySelector('body').classList.contains('logged-out')) {
-    console.log('offline');
-    selector = '.tweet';
+  console.log('offline');
+  selector = '.tweet';
 } else {
-    console.log('online');
-    selector = '[data-testid="tweet"]';
+  console.log('online');
+  selector = '[data-testid="tweet"]';
 }
+
 /*
-function which calls server for given node, and depending on the response, applies pre-defined action
+Depending on `option` sets classes to tweet nodes
  */
-let checkText = function(node) {
-    console.log('Sending Request');
-    let link = "https://api.optoutools.com/predict";
-    let xhr = new XMLHttpRequest();
-    xhr.open('POST', link, true);
-    xhr.setRequestHeader("Content-type", "application/json;charset=UTF-8");
-    xhr.withCredentials = true;
-    xhr.onreadystatechange = function (e) {
-        if (xhr.readyState !== 4) {
-            return;
-        }
-        if (xhr.status === 200) {
-            console.log("Response received as ", JSON.parse(xhr.response).predictions[0]);
-            if (JSON.parse(xhr.response).predictions[0]){
-                node.classList.add("processed-true");
-                let tweetText = node.querySelector(selector + ' > div ~ div > div ~ div');
-                styleTweet(tweetText, option);
-            }
-            else {
-                node.classList.add("processed-false");
-            }
-        } else {
-            console.log("Failed response", xhr);
-        }
-    };
-    xhr.send(
-        JSON.stringify({
-            texts: [node.innerText]
-        })
-    );
+const styleTweet = function (element, selectedOption) {
```

Comment:
Moved before the function definition before the first call (and added the missing `const`).